server/datastore/mysql/migrations/tables/20240202172022_CreateTableHostLockWipe.go

```diff
@@ -0,0 +1,32 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240202172022, Down_20240202172022)
+}
+
+func Up_20240202172022(tx *sql.Tx) error {
+	stmt := `
+		CREATE TABLE host_mdm_actions (
+			host_id INT UNSIGNED NOT NULL,
+			lock_ref VARCHAR(36) NULL,
+			wipe_ref VARCHAR(36) NULL,
```

Comment:
The ticket mentioned 3 boolean fields, did we decide to use a different approach? Might be worth adding a comment as to how those columns are expected to be used (i.e. what can be the lock_ref/wipe_ref values, and what role does `suspended` play?).

Also, did we consider adding those to `host_mdm` instead of a new table? I think the `host_mdm` one is mostly related to enrollment logic and may very well not be a good fit (@roperzh or @gillespi314 might know better), but just to make sure we considered the option.

Reply:
@mna  Ahh yeah, we had a call (Sarah, Robert, and I) on Friday where we discussed these things. 

>  I think the host_mdm one is mostly related to enrollment logic and may very well not be a good fit

Yeah, we considered it, but came to this exact conclusion

> The ticket mentioned 3 boolean fields, did we decide to use a different approach?

Yep, we went with a different approach. The `lock_ref` and `wipe_ref` columns will hold the UUIDs of the related commands/script executions that are locking or wiping the host. We realized we'd need a reference to these objects and that the references being there obviated the need for booleans.

`suspended` will track whether or not further actions on this host are suspended (e.g. [this state in the UI](https://www.figma.com/file/FCmmIh1y1DTzlRF5JsH3wh/%239949-%26-%239951-Remote-lock-and-remote-wipe-for-macOS%2C-Windows%2C-and-Linux?type=design&node-id=493-5858&mode=design&t=IoC8nm11MlntE7XW-4). We went with the name `suspended` to follow what's in the API updates: https://github.com/fleetdm/fleet/pull/16157/files#diff-7246bc304b15c8865ed8eaa205e9c244d0a0314e4bae60cf553dc06147c38b64R2018

I will add some comments to the code summarizing the above!